src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -388,12 +428,12 @@ private void ParseLocationFromAnalysisInfo(Result result)
                     // TODO: Confirm that the traces are ordered chronologically
                     // (so that we really do want to use the last one as the
                     // overall result location).
-                    PhysicalLocation = codeFlow.ThreadFlows[0].Locations.Last().Location?.PhysicalLocation
+                    PhysicalLocation = codeFlow.ThreadFlows[0].Locations.Last().Location?.PhysicalLocation.DeepClone()
```

Comment:
>DeepClone [](start = 107, length = 9)

Why clone here?

Reply:
Later I set a property value that differs between the two instances.

Reply:
The physicalLocation will get a region snippet later so we need a copied instance.

---
In reply to: [207320865](https://github.com/Microsoft/sarif-sdk/pull/981#discussion_r207320865) [](ancestors = 207320865)